src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -920,9 +1046,10 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
         public Task UnbanChatMemberAsync(
             ChatId chatId,
             int userId,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool onlyIfBanned = default
         ) =>
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.